teb_local_planner/launch/teb_view.rviz

```diff
@@ -5,7 +5,6 @@ Panels:
     Property Tree Widget:
       Expanded:
```

Comment:
I'm not sure you meant to commit your rviz config?

Reply:
Sure, I wanted to add `/teb_poses` and `/teb_markers` for the default visualization setting.

Reply:
Ok, there was just a bunch of changes. All good